### PR TITLE
Add a short documentation for `use`

### PR DESCRIPTION
**Pull request by weppos**
I was reading the "Getting started" guide and one of the things I noticed is that the section where `require`, `alias` and `import` are documented doesn't include any explanation for `use`. In fact, `use` is not mentioned at all in the entire getting started guide.

I know [`import`, `require` and `alias` are `SpecialForm`](http://elixir-lang.org/docs/v1.1/elixir/Kernel.SpecialForms.html) whereas `use` is [defined as `Kernel` macro](http://elixir-lang.org/docs/v1.1/elixir/Kernel.html#use/2), however it's not uncommon to see new users confused by this specific keyword: see [here](http://stackoverflow.com/questions/28491306/elixir-use-vs-import) and [here](http://stackoverflow.com/questions/29097918/elixir-what-does-the-use-keyword-do).

This branch takes bits and pieces from the answers above and the [`use/2` documentation](http://elixir-lang.org/docs/v1.1/elixir/Kernel.html#use/2), and adds a new section in the "getting started" to explain the purpose of `use`.

What do you think?

### CODE REVIEW

**Comment by josevalim** [comment]
THank you! @weppos the issue is that `use` is not necessarily lexically scoped as everything else. So maybe we could add this to the end of the chapter and start with something like you described: "Although not a directive, `use` is a macro tightly related to require, after all [current description]". What do you think?

**Comment by weppos** [comment]
Fine for me. Changing it now.

**Comment by weppos** [comment]
Done. Give it a look at let me know if it works for you.

File: getting-started/alias-require-and-import.markdown [hunk 1]
```diff
@@ -118,6 +118,7 @@ In the example above, the imported `List.duplicate/2` is only visible within tha
 
 Note that `import`ing a module automatically `require`s it.
 
+
```
Comment (josevalim) [inline getting-started/alias-require-and-import.markdown]: Extra line.

File: getting-started/alias-require-and-import.markdown [hunk 2]
```diff
@@ -192,3 +193,38 @@ end
 As we will see in later chapters, aliases also play a crucial role in macros, to guarantee they are hygienic.
 
 With this we are almost finishing our tour about Elixir modules. The last topic to cover is module attributes.
```
Comment (josevalim) [inline getting-started/alias-require-and-import.markdown]: This should be the last line of the chapter. :)

**Comment by josevalim** [comment]
I have added two more comments and we are good to go. :)

File: getting-started/alias-require-and-import.markdown (hunk 1 above)
Comment (weppos) [inline getting-started/alias-require-and-import.markdown]: It was probably added in the cut&paste. Removing it...

**Comment by weppos** [comment]
Here we go. Thanks for your feedback.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**